distributed/script/dserver.sh

```diff
@@ -1,5 +1,5 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
prevent BAD SUBSTITUTION errors: sh sometimes points to dash instead of bash (from what I understood)